src/mobilenet.jl

```diff
@@ -0,0 +1,304 @@
+      conv_bn((1, 1), hidden_planes, outplanes, identity; bias = false)...
+    )
+  else
+    invres = Chain(
+      conv_bn((1, 1), inplanes, hidden_planes, relu6; bias = false)...,
+      conv_bn((3, 3), hidden_planes, hidden_planes, relu6; bias = false, stride,
+               pad = 1, groups = hidden_planes)...,
+      conv_bn((1, 1), hidden_planes, outplanes, identity; bias = false)...
+    )
+  end
+
+  (stride == 1 && inplanes == outplanes) ? SkipConnection(invres, +) : invres
+end
+
+"""
+    mobilenetv2(width_mult; nclasses = 1000)
+
+Create a MobileNetv2 model.
+([reference](https://arxiv.org/abs/1801.04381)).
+
+# Arguments
+- `width_mult`: Controls the number of channels in each layer, with 1.0 being the original
+  model as detailed in the paper.
+- `nclasses`: the number of output classes
+"""
+function mobilenetv2(width_mult; nclasses = 1000)
+  configs = [
+    # t, c, n, s
+    [1, 16, 1, 1],
+    [6, 24, 2, 2],
+    [6, 32, 3, 2],
+    [6, 64, 4, 2],
+    [6, 96, 3, 1],
+    [6, 160, 3, 2],
+    [6, 320, 1, 1],
+  ]
+  
+  # building first layer
+  inplanes = _make_divisible(32 * width_mult, width_mult == 0.1 ? 4 : 8)
+  layers = []
+  append!(layers, conv_bn((3, 3), 3, inplanes, stride = 2))
+
+  # building inverted residual blocks
+  for (t, c, n, s) in configs
+    outplanes = _make_divisible(c * width_mult, width_mult == 0.1 ? 4 : 8)
+    for i in 1:n
+      push!(layers, invertedresidualv2(inplanes, outplanes, i == 1 ? s : 1, t))
+      inplanes = outplanes
+    end
+  end
+
+  # building last several layers
+  outplanes = (width_mult > 1.0) ? _make_divisible(1280 * width_mult, width_mult == 0.1 ? 4 : 8) : 1280
+
+  return Chain(Chain(layers...),
+               conv_bn((1, 1), inplanes, outplanes, relu6, bias = false)...,
+               AdaptiveMeanPool((1, 1)), flatten, Dense(outplanes, nclasses))
+end
+
+# Model definition for MobileNetv2
+struct MobileNetv2
+  layers
+end
+
+"""
+    MobileNetv2(width_mult::Float64 = 1.0; pretrain = false, nclasses = 1000)
+
+Create a MobileNetv2 model with the specified configuration.
+([reference](https://arxiv.org/abs/1801.04381)).
+Set `pretrain` to `true` to load the pretrained weights for ImageNet. 
+
+# Arguments
+- `width_mult`: Controls the number of channels in each layer, with 1.0 being the original
+  model as detailed in the paper.
+- `pretrain`: whether to load the pre-trained weights for ImageNet
+- `nclasses`: the number of output classes
+"""
+function MobileNetv2(width_mult::Float64 = 1.0; pretrain = false, nclasses = 1000)
+  layers = mobilenetv2(width_mult; nclasses)
+  pretrain && loadpretrain!(layers, string("MobileNetv2"))
+
+  MobileNetv2(layers)
+end
+
+@functor MobileNetv2
+
+(m::MobileNetv2)(x) = m.layers(x)
+
+backbone(m::MobileNetv2) = m.layers[1]
+classifier(m::MobileNetv2) = m.layers[2:end]
+
+# MobileNetv3
+
+"""
+Hard-Swish activation function
+([reference](https://arxiv.org/abs/1905.02244)).
+"""
+@inline h_swish(x) = x * hardσ(x)
```

Comment:
Makes more sense to make a PR to NNlib.jl with this.

Reply:
Have done so, will use that version once it gets merged

Reply:
https://github.com/FluxML/NNlib.jl/pull/376 takes care of this. But given that it's in `master` and not in the current release, keeping this line of code for now.

Reply:
Thank you so much!